content/docs/user-guide/experiment-management/managing-experiments.md

```diff
@@ -0,0 +1,68 @@
+# Managing Experiments
```

Comment:
I guess I'm the reason for this new section? I'm okay with leaving this in `Running experiments` depending on what others think. It does seem a bit disjointed as is. I think we need to at least link here from `Running experiments` so it's clear how to see and compare experiments.

Reply:
Moved